src/lib.rs

```diff
@@ -53,12 +53,11 @@
 //! ```compile_fail
 //!# use compact_arena::mk_tiny_arena;
 //! let idx = { mk_tiny_arena!(arena, 1); arena.add(1usize)) };
-//! assert!(core::mem::size_of_val(&a) == 4);
```

Comment:
Please don't remove this, change the value to 2 instead.

Reply:
there is no `a` in the scope.

Reply:
Sorry, my bad, that should be `idx`. Please try to remove the `compile-fail` to ensure you get a lifetime error.

Reply:
Anyway why should it be `idx`?

Reply:
Here is how it looks:

```
---- src/lib.rs -  (line 53) stdout ----
error[E0597]: `tag` does not live long enough
 --> src/lib.rs:55:13
  |
5 | let idx = { mk_tiny_arena!(arena); arena.add(1usize) };
  |     ---     ^^^^^^^^^^^^^^^^^^^^^^                   - `tag` dropped here while still borrowed
  |     |       |
  |     |       borrowed value does not live long enough
  |     borrow later stored here
```